src/Illuminate/Queue/SerializesModels.php

```diff
@@ -65,6 +65,12 @@ public function __serialize()
                 continue;
             }
 
+            $property->setAccessible(true);
+
+            if (! $property->isInitialized($this)) {
```

Comment:
How come this code doesn't crash on PHP 7.2 and 7.3. Are you sure the tests cover it?

Reply:
Don't you need to check the PHP version is at least 7.4 before calling that method?

Reply:
function __serialize() works only on PHP 7.4+

Reply:
Ah, yes. Thanks for clarifying.